client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.tsx

```diff
@@ -188,37 +189,22 @@ function CheckoutSummaryFeaturesList( props: {
 			) }
 			<CheckoutSummaryFeaturesListItem>
 				<WPCheckoutCheckIcon id="features-list-support-text" />
-				<SupportText hasPlanInCart={ hasPlanInCart } isJetpackNotAtomic={ isJetpackNotAtomic } />
+				{ translate( 'Customer support via email' ) }
 			</CheckoutSummaryFeaturesListItem>
 
 			{ ! hasPlanInCart && <CheckoutSummaryChatIfAvailable siteId={ siteId } /> }
 
-			{ showRefundText && (
-				<CheckoutSummaryFeaturesListItem>
-					<WPCheckoutCheckIcon id="features-list-refund-text" />
-					{ refundText }
-				</CheckoutSummaryFeaturesListItem>
-			) }
+			{ uniqueRefundableProducts.length &&
+				uniqueRefundableProducts.map( ( product ) => (
+					<CheckoutSummaryFeaturesListItem key={ product.uuid }>
+						<WPCheckoutCheckIcon id="features-list-refund-text" />
+						{ getRefundText( getRefundDays( product ), product.product_name, translate ) }
+					</CheckoutSummaryFeaturesListItem>
+				) ) }
 		</CheckoutSummaryFeaturesListWrapper>
 	);
 }
 
-function SupportText( {
-	hasPlanInCart,
-	isJetpackNotAtomic,
-}: {
-	hasPlanInCart?: boolean;
-	isJetpackNotAtomic?: boolean | null;
-} ) {
-	const translate = useTranslate();
-
-	if ( hasPlanInCart && ! isJetpackNotAtomic ) {
-		return <span>{ translate( 'Unlimited customer support via email' ) }</span>;
```

Comment:
This copy may be product specific. I would be more confident if we could ensure that this is not the case. @sirbrillig do you have an idea if we could remove this?

Reply:
Hm... the history of that block is quite old and has been updated several times. I don't have a particular opinion here, but you might consider checking with the Jetpack folks. I guess make sure that this issue is still fixed: https://github.com/Automattic/wp-calypso/issues/44793

Reply:
This was discussed briefly in https://github.com/Automattic/wp-calypso/pull/54210#issuecomment-1112293740 

It looks to me like the history behind the `! isJetpackNotAtomic` logic has to do with the fact that Jetpack sites would not receive live chat support and this string was previously `Email and live chat support`. See #44872 and #50943 for some history.

Regarding live chat, the `CheckoutSummaryChatIfAvailable` (in this same file) seems to be responsible for determining whether we offer live chat support or not for the products in the cart.

At this point, there doesn't seem to be any meaningful distinction between `Unlimited customer support via email` and `Customer support via email` from the customer's perspective.